src/server.js

```diff
@@ -0,0 +1,18 @@
+const express = require("express");
+const log = require("npmlog");
+
+export function getPort() {
+  return process.env.PORT ? parseInt(process.env.PORT, 10) : 3000;
```

Comment:
🎨 `Number.parseInt(process.env.PORT)` => https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/parseInt#Description

Reply:
please change this 😽